docs/en/loading/Kafka-connector-starrocks.md

```diff
@@ -11,99 +11,173 @@ The Kafka connector can seamlessly integrate with Kafka Connect, which allows St
+        key.converter.schemas.enable=true
+        value.converter.schemas.enable=false
+        # The absolute path of the starrocks-kafka-connector after extraction. For example:
+        plugin.path=/home/kafka-connect/starrocks-kafka-connector-1.0.3
+        ```
+    2. Run the Kafka Connect.
+        
+        ```BASH
+        CLASSPATH=/home/kafka-connect/starrocks-kafka-connector-1.0.3/* bin/connect-distributed.sh config/connect-distributed.properties
+        ```
+    
+2. Configure and create the Kafka connector. Note that in distributed mode, you need to configure and create the Kafka connector through the REST API. For parameters and descriptions, see [Parameters](#Parameters).
 
-        ```Shell
-        curl -i http://127.0.0.1:8083/connectors -H "Content-Type: application/json" -X POST -d '{
+      ```Shell
+      curl -i http://127.0.0.1:8083/connectors -H "Content-Type: application/json" -X POST -d '{
         "name":"starrocks-kafka-connector",
         "config":{
-            "connector.class":"com.starrocks.connector.kafka.SinkConnector",
-            "topics":"dbserver1.inventory.customers",
-            "starrocks.http.url":"192.168.xxx.xxx:8030,192.168.xxx.xxx:8030",
-            "starrocks.user":"root",
-            "starrocks.password":"123456",
-            "starrocks.database.name":"inventory",
-            "key.converter":"io.confluent.connect.json.JsonSchemaConverter",
-            "value.converter":"io.confluent.connect.json.JsonSchemaConverter"
-        }
+          "connector.class":"com.starrocks.connector.kafka.StarRocksSinkConnector",
+          "topics":"test",
+          "key.converter":"org.apache.kafka.connect.json.JsonConverter",
+          "value.converter":"org.apache.kafka.connect.json.JsonConverter",
+          "key.converter.schemas.enable":"true",
+          "value.converter.schemas.enable":"false",
+          "starrocks.http.url":"192.168.xxx.xxx:8030",
+          "starrocks.topic2table.map":"test:test_tbl",
+          "starrocks.username":"user1",
+          "starrocks.password":"123456",
+          "starrocks.database.name":"example_db",
+          "sink.properties.strip_outer_array":"true"
         }
-        ```
-
-3. Query data in the StarRocks table.
+      }'
+      ```
+      > **NOTICE**
+      >
+      > If the source data is CDC data, such as data in Debezium format, and the StarRocks table is a Primary Key table, you also need to [configure `transform`](#load-debezium-formatted-cdc-data) in order to synchronize the source data changes to the Primary Key table.
+
+#### Query StarRocks table
+
+Query the target StarRocks table `test_tbl`. The data is successfully loaded when the fowllowing result is returned.
+
+```mysql
+MySQL [example_db]> select * from test_tbl;
++------+-------------+
+| id   | city        |
++------+-------------+
+|    1 | New York    |
+|    2 | Los Angeles |
+|    3 | Chicago     |
++------+-------------+
+3 rows in set (0.01 sec)
+```
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/4196d01c-b70f-41a1-92a2-591cc249e0c4)
"MySQL [example_db]> select * from test_tbl;" is the query command. Only the part below the query command is the return result, so it's inappropriate to put "The data is successfully loaded when the fowllowing result is returned." above the query command together with the return result.
Move the "The data is successfully loaded when the fowllowing result is returned." sentence downwards right above the return result.